src/AnnotationDialog.js

```diff
@@ -329,17 +330,13 @@ class AnnotationDialog extends EventEmitter {
      * @return {void}
      * @protected
      */
-    addSortedAnnotations(annotations) {
+    sortAnnotationsList(annotations) {
         // Sort annotations by date created
-        const sorted = Object.keys(annotations).map((key) => annotations[key]);
-        sorted.sort((a, b) => {
+        this.annotations = Object.keys(annotations).map((key) => annotations[key]);
+        this.annotations.sort((a, b) => {
```

Comment:
you could do an implicit return here now that you aren't assigning any variables:
`this.annotations.sort((a, b) => new Date(a.created) - new Date(b.created);`